test/testdata/lsp/field_edits.1.rbupdate.document-symbols.exp

```diff
@@ -34,22 +34,22 @@
                     "kind": 14,
                     "range": {
                         "start": {
-                            "line": 1,
-                            "character": 10
+                            "line": 3,
+                            "character": 2
                         },
                         "end": {
-                            "line": 1,
-                            "character": 15
+                            "line": 3,
+                            "character": 7
```

Comment:
If you look at the corresponding ranges before the edit, you can see that the new values make a lot more sense than the previous values:

https://github.com/sorbet/sorbet/blob/4f8cdde4e605d0530c8eb24924b1ba71e80fbdf1/test/testdata/lsp/field_edits.rb.document-symbols.exp#L34-L43

The relevant parts of the before file:

https://github.com/sorbet/sorbet/blob/4f8cdde4e605d0530c8eb24924b1ba71e80fbdf1/test/testdata/lsp/field_edits.rb#L1-L3

The relevant parts of the after file:

https://github.com/sorbet/sorbet/blob/4f8cdde4e605d0530c8eb24924b1ba71e80fbdf1/test/testdata/lsp/field_edits.1.rbupdate#L1-L4

We added the comment line, so the locs should move down one line as well.